superdesk/publish/formatters/newsml_1_2_formatter.py

```diff
@@ -44,6 +44,9 @@ class NewsML12Formatter(Formatter):
     }
     ENCODING = "UTF-8"
 
+    name = None
```

Comment:
Any reason for `name = None` here?

Reply:
not really, I was probably testing something. will set it